lib/action_kit/client.rb

```diff
@@ -4,8 +4,8 @@ module Client
     def client(verb, path, params)
       Typhoeus::Request.send(
         verb,
-        "https://act.sumofus.org/rest/v1/#{path}/",
-        { userpwd: "#{ENV['AK_USERNAME']}:#{ENV['AK_PASSWORD']}" }.merge(params)
+        "#{Settings.ak_processor_url}/#{path}/",
```

Comment:
If this is pulled from an ENV var, don't forget to add it to all the AWS environments!